POK Updates

markmiltenburg opened the pull request:
Update calculator to account for Infantry/Mechs, new factions, and easy modifiers

markmiltenburg committed:
POK Updates

Update calculator to account for Infantry/Mechs, new factions, and easy modifiers

wholton commented:
The main repository is being updated by AlphaMouse and others, if you're interested in pushing your changes there as well.

markmiltenburg committed:
POK Modifiers Update

Adds additional modifiers to support:
- Naalu mech
- Nekro mech
- Winnu commander
- JolNar mech
- Mentak mech
- Mahact FS
- NRA Supercharge
- Nomad FS upgrade
- Nomad promissory
- Xxcha mech PDS
- Strike Wing Ambuscade
- Argent commander
- AFB extra dice
- Argent flagship
- Blitz
- NRA mech space combat
- L1 mech harrow
- Titans hero
- L1 commander
- Barony agent
- Sol agent
- NRA Flagship
- Sardakk mech
- Raid Formation
- Duranium mechs
- Winnu FS NRA mechs
- Waylay
